vulcan-next-sitemap.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  siteUrl: 'https://vulcan.next',
+  generateRobotsTxt: true,
+  //outDir: './out/', for static builds
+  exclude: ['/vns/*', '/login', '/profile', '/signup', '/admin', '/auth'],
```

Comment:
Can we instead have a whitelist approach with `include`? Vulcan app are often private app so it make more sense to whitelist what you want to see in the sitemap than what you don't want to see

Reply:
It doesn't look like that package supports include at the moment.

Reply:
The robots.txt file I'm thinking of is something like?
```allow: ['/','/docs/*'],```